app.js

```diff
@@ -16,4 +16,11 @@ export const getTopicsData = () => {
     return axios.get('https://nc-news-f67l.onrender.com/api/topics').then(({data}) => {
         return data
     })
+}
+
+export const getCommentsData = (article_id) => {
+    return axios.get(`https://nc-news-f67l.onrender.com/api/articles/${article_id}/comments`).then(({data}) => {
+        return data
+    })
+
```

Comment:
To make your API functions cleaner you can set a [base url](l2c.northcoders.com/courses/sd-notes/front-end/#sectionId=axios,step=creating-an-axios-instance) so that you aren't having to copy out the whole web address every time ✨ 

_p.s. the file name should be api.js rather than app.js_